src/Shared/Recruit.Vacancies.Client/Application/Validation/Fluent/CustomValidators/VacancyValidators/VacancyFluentValidationExtensions.cs

```diff
@@ -116,6 +117,27 @@ await trainingProviderSummaryProvider.GetAsync(trainingProvider.Ukprn.Value) !=
             });
         }
 
+        internal static IRuleBuilderInitial<Vacancy, Vacancy> TrainingProviderVacancyMustHaveEmployerPermission(this IRuleBuilder<Vacancy, Vacancy> ruleBuilder, IProviderRelationshipsService providerRelationshipService)

```

Comment:
In this class there is a method called `TrainingProviderMustExistInRoatp` and a call within a `if` condition like `await trainingProviderSummaryProvider.GetAsync(trainingProvider.Ukprn.Value) != null` where I believe the await call might not be evaluated before the `!= null` check. could you stick brackets around the await portion of this line. Otherwise I think it is checking != against a Task rather than the actual result of the async call.

Reply:
👍